src/main/java/align/GlobalOpt.java

```diff
@@ -171,25 +175,25 @@ public static void iterativeGlobalOpt(
 			{
 				int unaligned = tc.preAlign().size();
 				if ( unaligned > 0 )
-					System.out.println( "(" + new Date( System.currentTimeMillis() ) + "): pre-aligned all tiles but " + unaligned );
+					logger.info( "(" + new Date( System.currentTimeMillis() ) + "): pre-aligned all tiles but " + unaligned );
 				else
-					System.out.println( "(" + new Date( System.currentTimeMillis() ) + "): prealigned all tiles" );
+					logger.info( "(" + new Date( System.currentTimeMillis() ) + "): prealigned all tiles" );
 
 				TileUtil.optimizeConcurrently(
 						new ErrorStatistic( maxPlateauwidth + 1 ),  maxAllowedError, maxIterations, maxPlateauwidth, 1.0f,
 						tc, tc.getTiles(), tc.getFixedTiles(), numThreads );
 
-				System.out.println( "(" + new Date( System.currentTimeMillis() ) + "): Global optimization of " + tc.getTiles().size());
-				System.out.println( "(" + new Date( System.currentTimeMillis() ) + "):    Avg Error: " + tc.getError() + "px" );
-				System.out.println( "(" + new Date( System.currentTimeMillis() ) + "):    Min Error: " + tc.getMinError() + "px" );
-				System.out.println( "(" + new Date( System.currentTimeMillis() ) + "):    Max Error: " + tc.getMaxError() + "px" );
+				logger.info( "(" + new Date( System.currentTimeMillis() ) + "): Global optimization of " + tc.getTiles().size());
+				logger.info( "(" + new Date( System.currentTimeMillis() ) + "):    Avg Error: " + tc.getError() + "px" );
+				logger.info( "(" + new Date( System.currentTimeMillis() ) + "):    Min Error: " + tc.getMinError() + "px" );
+				logger.info( "(" + new Date( System.currentTimeMillis() ) + "):    Max Error: " + tc.getMaxError() + "px" );
 
 				// give some time for the output
 				try { Thread.sleep( 50 ); } catch ( Exception e) {}
 			}
 			catch (Exception e)
 			{
-				System.out.println( "Global optimization failed, please report this bug: " + e );
+				logger.error( "Global optimization failed, please report this bug: " + e );
 				e.printStackTrace();
```

Comment:
I'm not sure, but I suspect that the logger reports the stack trace anyway (at least if called with the `... {}", e)` syntax). In that case, the `e.printStackTrace()` can be removed.

Reply:
Addressed by [782bb33](https://github.com/PreibischLab/STIM/pull/8/commits/782bb3317a3d19a3fb727831fa72bd31264ca98a)